### ISSUE TITLE
coc.nvim error "Uncaught exception: Header must provide a Content-Length property"

### ISSUE THREAD

**Issue by bukephalos** [bscan/PerlNavigator#78]
I would like to use PerlNavigator with vim.
- coc.nvim is installed.
- followed the instructions to install PerlNavigator for other editors (git clone...)
- opened a trivial perl script with a syntax error (missing "my"): The error was marked and diagnostics shown (fine so far)
- moved the cursor to a subroutine call: "dosomething();" typed gd to go to the definition of the subroutine but got the following error message in the status line:
```
[coc.nvim]: Uncaught exception: Header must provide a Content-Length property
```
CocInfo indicates that PerlNavigator was started and then also shows the exception.

Perhaps I have a misconception about how "go to definition" works (I am just starting moving from pure editing to IDE functionality) but I think at least there should be a more helpful error message.

**Comment by bscan** [comment]
HI @bukephalos, I've been making quite a few changes this weekend, especially around the new v5.38 class feature. Unfortunately I broke go-to definition for non-vscode editors. I just pushed a fix, which is now available in version 0.6.0. Thanks for the bug report.

**Comment by bukephalos** [comment]
Works fine now, thanks!
